### DISCUSSION TITLE
When Does Testnet start?

### DISCUSSION THREAD

**Discussion by Saitama421** [pkgxdev/pkgx#935] [Help] [answered]
When Does Testnet start?

**Comment by Viacheslav86** [comment, marked as answer]
As I know testnet will be live on 21st of February

**Reply by dbentry** [reply]
Yes! Heard the same. Looking forward!

**Reply by Fidex123** [reply]
Pls wait for office announcement. The official date with be announced on discord and all other social platforms

**Comment by Fidex123** [comment]
Pls wait for official announcement. The official date with be announced on discord and all social platforms

**Comment by chris-setra** [comment]
Wednesday, February 21

**Comment by Kryptocapz** [comment]
Testnet will be live on 21st of February 2024.
Time has not been specified yet

**Comment by Ambrish211326** [comment]
testnet will be on 21st feb2024

**Comment by dangaoron** [comment]
Thanks for the comment info :) now we have a date to look forward to

**Comment by Edotamara26** [comment]
Yes, i'am wait for Tesnet

**Reply by miftah770** [reply]
testnet will be live on 21st of February

**Comment by hanriiver** [comment]
will Feb-21

**Comment by Oknessman** [comment]
Le testent c'est dans exactement 22jours
